script/deploy/contract-config.js

```diff
@@ -0,0 +1,18 @@
+const contractConfigData = {
```

Comment:
Can't we take these from `../networks/rinkeby.json`?

Reply:
resolved. now contract addresses are refered from rinkeby.json